style_guides/accessibility_guide.md

```diff
@@ -0,0 +1,108 @@
+# Accessibility (A11Y) Guide
+
+This document provides some technical guidelines how to prevent several common
+accessibility issues.
+
+## Interactive elements
+
+### Use `<button>` and `<a>`
+
+**TL;DR** *Use `<button>` and `<a>` instead of click listeners on other elements
+and style it whatever way you need.*
+
+If you want to make an element clickable, use a `<button>` or `<a>` element for it.
+Use a *button* whenever it causes an action on the current page, and an *a* if it
+navigates to a different page. You can use click listeners just fine on these elements.
```

Comment:
I would totally agree with you in that. But I think there might be some cases, where the "navigation" is actually implemented in JS (like dynamically determine parts of the URL) in which this would be valid. That's why I left it in there.